scripts/ci/tags.yaml

```diff
@@ -97,7 +97,11 @@ posix:
   files:
     - zephyr/lib/posix/
 
-# cbprintf:
-#    files:
-#        - zephyr/lib/os/cbprintf*
-#        - zephyr/lib/posix
+cellular:
+  files:
+    - nrf/lib/at*
+    - nrf/lib/*modem*
+    - nrf/lib/sms
```

Comment:
There are many more libraries with cellular dependency or that are used only or mainly with our cellular nRF91 chips. So is this list now based on some well-thought criteria?

Reply:
Great!
Can you specify them?
ie all relevant folders from NCS with source code related to cellular?